handlers/ocr-handler/server.py

```diff
@@ -115,12 +115,50 @@ def render_ocr():
         logging.debug("Sending response")
         return response
 
-    # Get text renderer data
-    text = 'The following ' + str(len(ocr_data['lines']))
-    text += ' lines were found in the image: '
-    for i, line in enumerate(ocr_data['lines']):
-        line_text = line['text'] + '\n'
-        text += line_text
+    # Text to be returned
+    text = ""
+
+    # Object detection data is present
+    od = 'ca.mcgill.a11y.image.preprocessor.objectDetection'
+    if od in preprocessors and len(preprocessors[od]['objects']) > 0:
+        object_data = preprocessors[od]
+        retmaining_text = ocr_data['lines']
+        remaining_objects = [{key: obj[key] for key
+                              in ['type', 'dimensions']} for
+                             obj in object_data['objects']]
+        text += "The following objects were detected: "
```

Comment:
I imagine this will eventually be worked into the other photo handler(s)?

Reply:
Yes just a proof of concept